clang_tidy/clang_tidy_test.bzl

```diff
@@ -40,6 +40,11 @@ def _clang_tidy_rule_impl(ctx):
     rule_conlyopts = _get_copts_attr(ctx, "conlyopts")
     rule_cxxopts = _get_copts_attr(ctx, "cxxopts")
 
+    files = ctx.attr.clang_tidy_resource_dir.files.to_list()
+    if files:
+        directory = files[0].path.split("/include/")[0]
```

Comment:
this is ugly but i think we'd have to use skylib directly to access a better api from the provider to improve this